packages/ra-ui-materialui/src/list/ListView.tsx

```diff
@@ -144,7 +143,6 @@ ListView.propTypes = {
     setSort: PropTypes.func,
     showFilter: PropTypes.func,
     title: TitlePropType,
-    total: PropTypes.number,
```

Comment:
Nice catch!

Can you also remove the other propTypes that shouldn't be there?

Thanks